packages/cozy-client/src/CozyClient.js

```diff
@@ -690,9 +690,21 @@ class CozyClient {
     if (stackClient) {
       this.stackClient = stackClient
     } else {
+      const options = {
+        ...this.options,
+        onTokenRefresh: token => {
+          this.emit('tokenUpdated', token)
```

Comment:
s/tokenUpdated/tokenRefreshed

Let's not introduce new words so that it is easier to transition ;)

Reply:
I think we should make it harder to get the token as it should be internal to cozyClient.

Something like `this.emit('tokenRefreshed', { dangerousToken: token })` ?

Reply:
I remove token to force the developer to go through the cozyClient to retrieve the token if he needs it.